README.md

```diff
@@ -379,6 +385,80 @@ logger.addHandler(handler)
 logger.removeHandler(handler)
 ```
 
+## Thundra Foresight
+
+Foresight is a project powered by Thundra agent to show every detail for test runs. For know, it only supports pytest. Although Its automatically enabled after installing Thundra as 'pip install thundra', you need to set THUNDRA_APIKEY and THUNDRA_AGENT_TEST_PROJECT_ID as described below. 
+
+### Activating Thundra Foresight
+
+- Firstly, setting up environment for Thundra. There are three ways to do it.
+
+    1. Setting THUNDRA_APIKEY and THUNDRA_AGENT_TEST_PROJECT_ID as environment variable.
+
+    ```sh
+    export THUNDRA_APIKEY=<your_apikey>
+    export THUNDRA_AGENT_TEST_PROJECT_ID=<your_test_project_id>
+    ```
+
+    2.  "thundra_apikey " and "thundra_agent_test_project_id" in you .env file. Then, loaded them in conftest.py file like:
+
+    ```.env
+        thundra_apikey =<your_apikey>
+        thundra_agent_test_project_id = <your_test_project_id>
+    ```
+
+    ```conftest.py
+        # This method requires python >= 3.5
+        from pathlib import Path
+
+        from dotenv import load_dotenv
+
+        env_path = Path(<your_env_file_name>)
+        load_dotenv(dotenv_path=env_path)
+    ```
+
+    3. Importing Thundra in conftest file and configure it.
+
+    ```conftest.py
+        import thundra
+
+        thundra.configure(
+            options={
+                "config": {
+                    "thundra.apikey": <your_apikey>,
+                    "thundra.agent.test.project.id": <your_test_project_id>
+                }
+            }
+        )
+    ```
+
+### Deactivating Thundra Foresight
+
+- There are two ways to deactivate Thundra Foresight for pytest:
```

Comment:
`There are two ways` is written but there are three parts in below

Reply:
fixed.